### PR TITLE
Minimalist ad experience post

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

**Review by ericholscher — COMMENTED** [review]
This is a good start. Definitely could use some editing to make points clearer and talk more directly. 

I'm not really sure if it's going to get much in terms of SEO. Would be good to define the audience of this post a bit more -- is it mainly for people Googling "Why do some sites have so many ads?".

I think we can ship this one, but we might also consider another version that talks more explicitly about promoting the benefits of our approach, and has more positive examples. Something promoting the value of minimalist ads, with a screenshot of minimalist examples, and why building longer term value is :100:. This would perhaps capture people looking for "Best way to monetize respectfully with ads" or similar.

**Comment by davidfischer** [comment]
I pushed an update that responds to the feedback. Specifically it:

- Has an example screenshot of a more minimalist experience
- Uses the term "complementing" rather than minimalist.
- Talks about ads that don't get in the way
- Highlights that our network focuses on high-quality sites rather than drive-by sites

**Review by ericholscher — APPROVED** [review]
👍

File: content/posts/2024-minimalist-ads.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Title: Why Some Sites Show More Ads Than Content
+Date: April 9, 2024
```
Comment (davidfischer) [inline content/posts/2024-minimalist-ads.md:2]:
```suggestion
Date: April 30, 2024
```

**Comment by davidfischer** [comment]
I'm aiming for next Tuesday (4/30) to go live with this one.

File: content/posts/2024-minimalist-ads.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+Title: Why Some Sites Show More Ads Than Content
+Date: April 30, 2024
```
Comment (davidfischer) [inline content/posts/2024-minimalist-ads.md:2]:
```suggestion
Date: May 2, 2024
```

**Pull request merged**